backend/server.js

```diff
@@ -15,6 +14,12 @@ app.use(cors());
 
 app.listen(PORT, () => console.log("Server running on PORT " + PORT));
 
+app.get("/", (req, res) => {
+  res.status(200).json({
+      message: "Testing whether the API works"
+  });
+});
+
```

Comment:
This is only a sample route you can use to confirm that your API starts successfully

Reply:
I got it.